package.json

```diff
@@ -149,14 +141,30 @@
         }
       ],
       "@semantic-release/changelog",
-      [
-        "@semantic-release/npm",
-        {
-          "pkgRoot": "dist"
-        }
-      ],
+      "@semantic-release/npm",
       "@semantic-release/github",
       "@semantic-release/git"
     ]
+  },
+  "scripts": {
+    "clean": "aegir clean",
+    "lint": "aegir lint",
+    "build": "aegir build",
+    "release": "aegir release",
+    "test": "aegir test -t node -t browser -t webworker -t electron-main",
+    "test:ts": "npm run test --prefix test/ts-use",
```

Comment:
I wasn't sure what to do with this, since we don't test ts integration like this in libp2p/ipfs anywhere else (yet).

Just tagged it on to the end of `test:node` for now - if it's useful we can add a block to the `js-test-and-release.yml` action?  Might be preferable.

Reply:
It seems the test command in `ts-use` uses an argument format that means it doesn't work on Windows.

Reply:
Having the TS tests be par of `js-test-and-release.yml` would be great. I'm not sure how they should be approached yet though.

Would running the tests as TS but having the TS tests import JS tests work?

Reply:
It's worth a shot, though the tests are compiled to js before execution so my brain turns inside out a little bit.